NuGet.config

```diff
@@ -3,5 +3,9 @@
   <packageSources>
     <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
     <add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+    <add key="BuildTools" value="https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
```

Comment:
Are you just trying to add MSBuild? Or are there other dependencies? MSBuild is pushing packages to  https://dotnet.myget.org/F/msbuild/api/v3/index.json now